lib/api/v3/work_packages/work_package_deleted_representer.rb

```diff
@@ -0,0 +1,62 @@
+# -- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) 2010-2023 the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+# ++
+
+module API
+  module V3
+    module WorkPackages
+      class WorkPackageDeletedRepresenter < ::API::Decorators::Single
+        include TimestampedRepresenter
+        attr_accessor :timestamps, :query
+
+        def initialize(model, current_user:, embed_links: false, timestamps: nil, query: nil)
+          @query = query
+          @timestamps = timestamps || query.try(:timestamps) || []
+
+          super(model, current_user:, embed_links:)
+        end
+
+        def self_v3_path(*)
+          api_v3_paths.work_package(represented.id, timestamps:)
+        end
+
+        self_link title_getter: ->(*) {}
+
```

Comment:
That representer has no caching currently. It is a very bare bone representer, not inheriting from the WorkPackageRepresenter. Because of that, I don't think the option needs to be added.